Projects/Ripple-Lights-App.md

```diff
@@ -0,0 +1,31 @@
+# RippleLights
+
+The RippleLights application relies on your development talents to create
+a mesmerizing light display. Your task is to draw seven colored circles 
+in a row and based on a timer change the intensity of each each circle. When
+a circle is brightened it's predecessor returns to its normal intensity. 
+
+This simulates the effect of a string of rippling lights, similar to the ones
+displayed during the Christmas Holidays.
```

Comment:
I'll highlight the *Christmas Holidays* keyword and effect. This would be much clearer in my opinion. Everyone know how Christmas lights work 😄

Reply:
Renamed app to 'ChristmasLights' and updated links to make it easier for users to understand the apps purpose.